pyfolio/utils.py

```diff
@@ -129,8 +131,8 @@ def get_symbol_rets(symbol):
         if symbol == 'SPY':
             try:
                 rets.to_hdf(filepath, 'df')
-            except:
-                pass
+            except Exception as e:
+                warnings.warn('Failed to save SPY locally: %s' % e)
```

Comment:
This should print the path it tried to save to, since the most likely cause of the error is that the path doesn't exist or doesn't have appropriate permissions, both of which can be diagnosed easily with access to the target path.